apps/framework-cli/src/utilities/package_managers.rs

```diff
@@ -98,7 +112,39 @@ pub fn link_sdk(
     }
 
     let output = command.output()?; // We should explore not using output here and instead using spawn.
-    println!("{}", String::from_utf8(output.stdout).unwrap());
+    match String::from_utf8(output.stdout) {
+        Ok(val) => {
+            debug!("{}", val);
+        }
+        Err(e) => {
+            error!("Error: {:?}", e);
+        }
+    }
```

Comment:
I don't really like 8 lines of duplicated code.

Reply:
@phiSgr how are those lines duplicated code? The intent is to output status to the logger.

Reply:
The same lines are in `install_packages`,`run_build`, and `link_sdk`.

Reply:
Noted.  I image that's the case in other parts of the code base.